src/sdk/base/mediastream-factory.js

```diff
@@ -198,11 +202,9 @@ export class MediaStreamFactory {
       if (constraints.video.resolution &&
           constraints.video.resolution.width &&
           constraints.video.resolution.height) {
-        mediaConstraints.video.width = Object.create({});
-        mediaConstraints.video.width.exact =
+        mediaConstraints.video.width = 
```

Comment:
It changes SDK's behavior. `exact` is what we expected.

Reply:
But when use the `exact` value, error will happen
```
TypeError: Failed to execute 'getDisplayMedia' on 'MediaDevices': exact constraints are not supported
```

Reply:
Change to use 'number' only when share screen is needed.